deploy/roles/elasticsearch-config/tasks/main.yml

```diff
@@ -153,3 +153,13 @@
     name: elasticsearch
     state: started
     enabled: yes
+
+- name: Allow admins to read logs
+  acl:
+    path: "{{ item }}"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rx
+    state: present
+  with_items:
+    - /var/log/elasticsearch
```

Comment:
@stirlingv This should have `recursive: yes` to ensure that any existing log files are readable, and it should have a separate section for setting the default, as with the other roles.